arch/arm/configs/msm8974-perf_defconfig

```diff
@@ -450,6 +450,7 @@ CONFIG_ANDROID_BINDER_IPC=y
 CONFIG_ASHMEM=y
 CONFIG_ANDROID_LOGGER=y
 CONFIG_ANDROID_RAM_CONSOLE=y
+CONFIG_ANDROID_PERSISTENT_RAM=y
```

Comment:
We use cyanogen_cancro_defconfig, and this change is already present there. Hence this is not needed